api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriCohortQueries.java

```diff
@@ -31,6 +31,8 @@
   @Autowired private GenericCohortQueries genericCohortQueries;
 
   @Autowired private HivCohortQueries hivCohortQueries;
```

Comment:
`hivCohortQueries` has no more usages in this class.

Reply:
That is correct. Already removed with a new commit